kalite/distributed/management/commands/retrievecontentpack.py

```diff
@@ -140,3 +143,60 @@ def extract_content_db(zf, lang):
     with open(content_db_path, "wb") as f:
         dbfobj = zf.open("content.db")
         shutil.copyfileobj(dbfobj, f)
+
+
+def extract_subtitles(zf, lang):
+    SUBTITLE_DEST_DIR = get_subtitle_path(lang_code=lang)
+    SUBTITLE_ZIP_DIR = "subtitles/"
+
+    ensure_dir(SUBTITLE_DEST_DIR)
+
+    subtitles = (s for s in zf.namelist() if SUBTITLE_ZIP_DIR in s)
+
+    for subtitle in subtitles:
+        # files inside zipfiles may come with leading directories in their
+        # names, like subtitles/hotdog.vtt. We'll only want the actual filename
+        # (hotdog.vtt) when extracting as that's what KA Lite expects.
+
+        subtitle_filename = os.path.basename(subtitle)
+        subtitle_dest_path = os.path.join(SUBTITLE_DEST_DIR, subtitle_filename)
+
+        subtitle_fileobj = zf.open(subtitle)
+
+        with open(subtitle_dest_path, "w") as dest_fileobj:
+            shutil.copyfileobj(subtitle_fileobj, dest_fileobj)
+
+
+def extract_assessment_items(zf):
+    assessment_zip_dir = "assessment_resources/"
+
+    try:
+        channel = zf.read("channel.name")
```

Comment:
@benjaoming is this still valid? Note that there is no `channel.name` in the content packs, so this will always be empty, and channel will always be `""`. Should we specify the channel somehow?